pkg/router/functionHandler.go

```diff
@@ -319,6 +331,7 @@ func (roundTripper RetryingRoundTripper) RoundTrip(req *http.Request) (*http.Res
 			return resp, nil
 		} else if i >= roundTripper.funcHandler.tsRoundTripperParams.maxRetries-1 {
 			// return here if we are in the last round
+			//Check for timeout error first
```

Comment:
A blank should follow the `//`. (Here and some other places)

Reply:
This comment was to be removed. Forgot to remove it.